api/spec/smithy/model/operations/GetClusterLogEvents.smithy

```diff
@@ -0,0 +1,43 @@
+namespace parallelcluster
+
+@suppress(["MissingPaginatedTrait", "PaginatedTrait", "InputOutputStructureReuse"])
```

Comment:
why do we need to suppress "MissingPaginatedTrait", "PaginatedTrait"? isn't this properly paginated?

for the ones ew need to suppress can you leave a comment in the code explaining why.

Same for all other operations in this PR